modules/misc/xdg.nix

```diff
@@ -14,6 +14,7 @@ let
   defaultCacheHome = "${config.home.homeDirectory}/.cache";
   defaultConfigHome = "${config.home.homeDirectory}/.config";
   defaultDataHome = "${config.home.homeDirectory}/.local/share";
+  defaultRuntimeHome = "${config.home.homeDirectory}/.local/run";
```

Comment:
Unfortunately it is not possible to allow the user to edit the runtime directory path. It should be set by the user login process. So please revert the changes in this file.